CD_CODE/MAIN.m

```diff
@@ -92,6 +138,26 @@
 Aout            =   A*ones(Npts,1);
 Fcoef           =   -99*ones(Npts,1);
 
+%SURFACE MODEL (PLATE MODEL) GEOMETRY
+if obj_type == 4 %geometry file
+    
+    %open geometry and convert to vertex array
+    hf0       	=   figure('visible','off');
+    ViewDir     =   [Phi,Theta];%view direction
+    geometry_wizard(fnamesurf,'TRIprops.txt',0,1,ViewDir)
```

Comment:
Could we make this an input option to the function or dependent on another file location? Our server deployment can't write to the local directory, so this will error out if I don't manually change it later.

Maybe something like getting the filepath of `fnamesurf` and put it in front of TRIprops.txt so that you'll write the file out to the same location the original file is coming from? I'll likely be sending in absolute paths.
https://www.mathworks.com/help/matlab/ref/fileparts.html

```matlab
dir_path = fileparts(fnamesurf)(0)
tri_file = fullfile(dir_path, 'TRIprops.txt')
```
I think that will maintain the current directory path as you have it right now too...?